react-delegationdashboard/src/config.internal.ts

```diff
@@ -18,39 +18,20 @@ export const yearSettings = [
   { year: 11, maximumInflation: 0.0 },
 ];
 
-export const networks: NetworkType[] = [
-  {
-    default: false,
-    id: 'mainnet',
-    name: 'Mainnet',
-    theme: 'light',
-    egldLabel: 'EGLD',
-    walletAddress: 'https://wallet.elrond.com/dapp/init',
-    apiAddress: 'https://api.elrond.com',
-    gatewayAddress: 'https://gateway.elrond.com',
-    explorerAddress: 'http://explorer.elrond.com',
-    delegationContract: 'erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqhllllsajxzat',
-    auctionContract: 'erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqplllst77y4l',
-    stakingContract: 'erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqllls0lczs7',
-  },
-  {
-    default: true,
-    id: 'testnet',
-    name: 'Testnet',
-    theme: 'testnet',
-    egldLabel: 'xEGLD',
-    walletAddress: 'http://internal-wallet.elrond.com/dapp/init',
-    apiAddress: 'http://35.210.167.16:8080',
-    gatewayAddress: 'https://internal-gateway.elrond.com',
-    explorerAddress: 'http://internal-explorer.elrond.com/',
-    delegationContract: 'erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqhllllsajxzat',
-    auctionContract: 'erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqplllst77y4l',
-    stakingContract: 'erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqllls0lczs7',
-  },
-];
+export const network: NetworkType = {
```

Comment:
I don't think internal config should be public :)

Reply:
done